test/lightning_web/live/dashboard_live_test.exs

```diff
@@ -262,6 +277,47 @@ defmodule LightningWeb.DashboardLiveTest do
       assert project_last_activities_from_html ==
                projects_sorted_by_last_activity_desc
     end
+
+    test "Toggles the arcade banner", %{conn: conn} do
+      {:ok, view, _html} = live(conn, ~p"/projects")
+
+      # Assert that the banner is initially expanded
+      assert view
+             |> has_element?("#arcade-banner-content[class~='max-h-[500px]']")
+
+      # Click to collapse the banner
+      view
+      |> element("button[phx-click='toggle-arcade-banner']")
+      |> render_click()
+
+      # Assert that the banner is now collapsed
+      refute view
```

Comment:
@stuartc isn't this the reason why the tests were still passing ? Is it `~=` or `=~` ?

Reply:
Ah no, it's correct, I didn't update the templates properly - just pushed. Thanks! 👍